wagtail/wagtail
# Remove unused search-bar and button-filter styles

Those styles were used before the legacy Wagtail 3.0 / 4.0 header changes, they no longer do anything:

- `button-filter` isn’t present anywhere in the UI.
- There was only one use of the `required` CSS class, but in a `_file_field_as_li.html` template that’s only used on document editing forms (not inside a search-bar).

I’ve additionally removed said template, as both its addition of a `required` class and `error` don’t do anything – those two characteristics of the field are now set within `field.html`. There’s more related forms cleanup to do in #9031.

_Please check the following:_

    -   [x] **Please list the exact browser and operating system versions you tested**: Chrome 108 macOS 13 only
    -   [x] **Please list which assistive technologies [^3] you tested**: None

To test this, I’d recommend:

- Code-search `button-filter` to confirm it’s not present anywhere
- Code-search `search-bar` to confirm it’s no longer present anywhere
- Load the document edit form `http://localhost:8000/admin/documents/edit/1/` and check the appearance of the file field
